streams/src/main/java/org/apache/kafka/streams/errors/BrokerNotFoundException.java

```diff
@@ -23,6 +23,7 @@
  * @see org.apache.kafka.streams.StreamsConfig
  */
 @SuppressWarnings("unused")
+@Deprecated(since = "4.2", forRemoval = true)
```

Comment:
We should update the JavaDocs with `@deprecated` tag:
```
@deprecated Since 4.2 and should not be used any longer.
```

Reply:
updated javadoc